x-pack/plugins/security_solution/public/onboarding/components/onboarding_body/cards/siem_migrations/ai_connector/ai_connector_card.tsx

```diff
@@ -48,18 +40,11 @@ export const AIConnectorCard: OnboardingCardComponent<AIConnectorCardMetadata> =
   const canCreateConnectors = checkCompleteMetadata?.canCreateConnectors;
 
   return (
-    <OnboardingCardContentPanel
-      style={{
-        paddingTop: 0,
-        ...(isDarkMode && { backgroundColor: euiTheme.colors.lightestShade }),
-      }}
-    >
+    <OnboardingCardContentPanel>
       {canExecuteConnectors ? (
```

Comment:
nit: It has a quick flash showing `MissingPrivilegesCallOut` when `checkCompleteMetadata` is available yet.


https://github.com/user-attachments/assets/0c277cb3-6218-4997-a7eb-efc45e8e360c

Reply:
Yup, good catch. I'll address that in the following PR 👍

Reply:
cc: @agusruidiazgd This might also happen in the assistant card